support-frontend/typescript-errors.json

```diff
@@ -1658,6 +1612,10 @@
         {
             "path": "assets/pages/paper-subscription-checkout/helpers/orderSummaryText.ts",
             "message": "Property 'NoProductOptions' is missing in type '{ Saturday: string; SaturdayPlus: string; Sunday: string; SundayPlus: string; Weekend: string; WeekendPlus: string; Sixday: string; SixdayPlus: string; Everyday: string; EverydayPlus: string; }' but required in type 'Record<ProductOptions, string>'."
+        },
+        {
+            "path": "assets/pages/subscriptions-redemption/subscriptionsRedemption.tsx",
+            "message": "Property 'billingPeriod' is missing in type '{ countryGroupId: CountryGroupId; marketingConsent: Element; }' but required in type 'Omit<Pick<PropTypes, \"paymentMethod\" | \"productPrices\" | \"billingPeriod\" | \"marketingConsent\" | \"countryGroupId\" | \"currencyId\" | \"includePaymentCopy\"> & Partial<...> & Partial<...>, \"paymentMethod\" | ... 1 more ... | \"includePaymentCopy\">'."
```

Comment:
Is this an error we need to fix?

Reply:
I've now made the billing period argument optional for the patrons `getPromotion` function. This is one example where we use the a related component but don't need to pass through the billing period!